Game/Source/GDKTestGyms/AUptimeGameMode.cpp

```diff
@@ -2,433 +2,104 @@
 
 #include "AUptimeGameMode.h"
 
-#include "AIController.h"
-#include "BehaviorTree/BlackboardComponent.h"
-#include "BenchmarkGymNPCSpawner.h"
-#include "DeterministicBlackboardValues.h"
 #include "Engine/World.h"
-#include "EngineClasses/SpatialNetDriver.h"
-#include "EngineUtils.h"
-#include "GameFramework/Character.h"
-#include "GameFramework/PlayerStart.h"
-#include "GeneralProjectSettings.h"
 #include "Interop/SpatialWorkerFlags.h"
-#include "Misc/CommandLine.h"
-#include "Misc/Crc.h"
 
 DEFINE_LOG_CATEGORY(LogUptimeGymGameMode);
 
 namespace
 {
-	const FString UptimePlayerDensityWorkerFlag = TEXT("player_density");
-	const FString UptimePlayerDensityCommandLineKey = TEXT("PlayerDensity=");
-
-	const FString UptimeSpawnColsWorkerFlag = TEXT("spawn_cols");
-	const FString UptimeSpawnColsCommandLineKey = TEXT("-SpawnCols=");
-	
-	const FString UptimeSpawnRowsWorkerFlag = TEXT("spawn_rows");
-	const FString UptimeSpawnRowsCommandLineKey = TEXT("-SpawnRows=");
-
-	const FString UptimeWorldWidthWorkerFlag = TEXT("zone_width");
-	const FString UptimeWorldWidthCommandLineKey = TEXT("-ZoneWidth=");
-
-	const FString UptimeWorldHeightWorkerFlag = TEXT("zone_height");
-	const FString UptimeWorldHeightCommandLineKey = TEXT("-ZoneHeight=");
-
-	const FString UptimeEgressSizeWorkerFlag = TEXT("egress_test_size");
-	const FString UptimeEgressSizeCommandLineKey = TEXT("-EgressTestSize=");
+	const FString EgressSizeWorkerFlag = TEXT("egress_test_size");
+	const FString EgressSizeCommandLineKey = TEXT("-EgressTestSize=");
 	
-	const FString UptimeEgressFrequencyWorkerFlag = TEXT("egress_test_frequency");
-	const FString UptimeEgressFrequencyCommandLineKey = TEXT("-EgressTestFrequency=");
+	const FString EgressFrequencyWorkerFlag = TEXT("egress_test_frequency");
+	const FString EgressFrequencyCommandLineKey = TEXT("-EgressTestFrequency=");
 	
-	const FString UptimeCrossServerSizeWorkerFlag = TEXT("cross_server_size");
-	const FString UptimeCrossServerSizeCommandLineKey = TEXT("-CrossServerSize=");
+	const FString CrossServerSizeWorkerFlag = TEXT("cross_server_size");
+	const FString CrossServerSizeCommandLineKey = TEXT("-CrossServerSize=");
 
-	const FString UptimeCrossServerFrequencyWorkerFlag = TEXT("cross_server_frequency");
-	const FString UptimeCrossServerFrequencyCommandLineKey = TEXT("-CrossServerFrequency=");
+	const FString CrossServerFrequencyWorkerFlag = TEXT("cross_server_frequency");
+	const FString CrossServerFrequencyCommandLineKey = TEXT("-CrossServerFrequency=");
 } // anonymous namespace
 
 AUptimeGameMode::AUptimeGameMode()
-	: bInitializedCustomSpawnParameters(false)
-	, SpawnCols(2)
-	, SpawnRows(1)
-	, ZoneWidth(1000000.0f)
-	, ZoneHeight(1000000.0f)
-	, TestDataSize(0)
-	, TestDataFrequency(0)
+	: EgressSize(0)
+	, EgressFrequency(0)
 	, CrossServerSize(0)
 	, CrossServerFrequency(0)
-	, NumPlayerClusters(1)
-	, PlayersSpawned(0)
-	, NPCSToSpawn(0)
 {
+	PercentageSpawnPointsOnWorkerBoundaries = 0.0f;
```

Comment:
Initialise this in the class it's declared in, not here

Reply:
So this is the derived class (AUptimeGameMode) overriding the value. Is there a nice way to do this?